compiler/setup.py

```diff
@@ -12,9 +14,15 @@ def readme():
     """
     return readme_short
 
+
+def quilt_version():
+    pkgdir = os.path.split(os.path.abspath(__file__))[0]
```

Comment:
`pkgdir = os.path.dirname(__file__)`

I don't think you even need absolute path; relative ones work, too.

Reply:
Nope, I it's not necessary in this case, just done habitually.  Working with absolute paths isn't a bad habit, though, and it doesn't really obscure what's happening here.  Changed to your preference, anyway, as there's no need for it here.